test/server.js

```diff
@@ -195,6 +195,37 @@ describe('Server', () => {
         });
     });
 
+    describe('caches', () => {
+
+        it('shows cache segments within default cache provision', () => {
+
+            const server = Hapi.server();
+            const cache = server.cache({ segment: 'test', expiresIn: 1000 });
+
+            expect(server.caches._default.shared).to.be.equal(false);
+            expect(server.caches._default.segments.test).to.be.equal(cache);
```

Comment:
This assertion is a bit simpler and stricter.
```suggestion
            expect(server.caches._default.segments.test).to.shallow.equal(cache);
```

Reply:
Nice, I like the explicitness of the `shallow` modifier. Will update.